examples/calculations/Angle_to_Direction.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2019 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""
+Angle to Direction
+==================
+
+Demonstrate how to convert angles to direction strings.
+
+The code below shows how to convert angles into directional text.
+It also  demonstrates the function's flexibility.
+"""
+import metpy.calc as mpcalc
+from metpy.units import units
+
+###########################################
+# Create a test value of an angle
+angle_deg = 70 * units('degree')
+print(angle_deg)
+
+###########################################
+# Now throw that angle into the function to
+# get the corresponding direction
+dir_str = mpcalc.angle_to_dir(angle_deg)

```

Comment:
Sorry I didn't catch this with the last review!
```suggestion
dir_str = mpcalc.angle_to_direction(angle_deg)
```